appformer-kogito-bridge/src/test/java/org/appformer/kogito/bridge/client/context/KogitoChannelTest.java

```diff
@@ -0,0 +1,35 @@
+/*
+ * Copyright 2020 Red Hat, Inc. and/or its affiliates.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package org.appformer.kogito.bridge.client.context;
+
+import org.junit.Test;
+
+import static org.junit.Assert.assertEquals;
+
+public class KogitoChannelTest {
+
+    @Test
+    public void withNameTest() {
+        assertEquals(KogitoChannel.GITHUB, KogitoChannel.withName("GitHub"));
```

Comment:
Please cover also `ONLINE`, `VSCODE`, `DEFAULT`

Reply:
@jomarko can we fix this in a later JIRA? @jesuino is on PTO and I don't want to block the PR.

Reply:
https://issues.redhat.com/browse/KOGITO-866